db/migrate/20160203212001_add_card_token_to_order.rb

```diff
@@ -1,5 +0,0 @@
-class AddCardTokenToOrder < ActiveRecord::Migration
-  def change
-    add_column :orders, :card_token, :string
-  end
-end
```

Comment:
Yea - just bad behavior because noone else is contributing to this project. Won't do it again :)